src/service/file_manager/file_manager.service.ts

```diff
@@ -23,11 +36,39 @@ export class FileManagerService {
     return folder;
   }
 
-  public async getContent(user: User, location: string): Promise<{ files: Array<File>; folders: Array<Folder> }> {
-    const prefix = `organization/${user.id}/user/${user.id}`;
-    const wLocation = location.endsWith('/') ? location : `${location}/`;
-    const userLocation = path.join(prefix, wLocation);
+  public async uploadFiles(user: User, body: UploadFileRequest): Promise<UploadFileResponse> {
+    const links = Promise.all(
+      body.fileInfos.map(async (fileInfo) => {
+        const file = FileFactory.from({
+          userId: user.id,
+          organizationId: user.id,
+          folder: body.location,
+          filename: fileInfo.name,
+          size: fileInfo.sizeL,
+        });
+        await this.storageService.upload(file.key);
+        return { name: fileInfo.name, url: (await this.storageService.getSignedPostUrl(fileInfo.name, file.size)).url };
```

Comment:
It won't work at all. At first you don't need to upload this file. Why do we need uploading an EMPTY here? Additionally, you send a SignedPostUrl for absolutely another key. Also, please make more variables here, it's hard to read this map.